### PR TITLE
Print built image name at the end of function create

### LINKED ISSUE #1201: Make image name easier to figure out after build
Since #1105 developers can call `function create` without specifying `--image`

Developers should have an easy way to see the actual image name produced from the `--verbose` output of the `function create`, and maybe even from the non-verbose output or perhaps from a variant of `service status` or `service list`.

As a starting point, I suggest adding the image name to the last line of `function create` e.g.

```
riff function create: built image xxxx/xx for service xx. 
```

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes #1201

### CODE REVIEW

File: pkg/riff/commands/function_test.go [hunk 1]
```diff
@@ -462,3 +462,21 @@ var _ = Describe("The riff function build command", func() {
...
+							Container: corev1.Container{
+								Image: img,
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+}
```
Comment (fbiville) [inline pkg/riff/commands/function_test.go]: nice

**Comment by fbiville** [comment]
Merged in 022e09db1895c8d99ec5bce052560a50b11f3cac

File: pkg/riff/commands/function.go [hunk 2]
```diff
@@ -114,7 +114,9 @@ func FunctionCreate(buildpackBuilder core.Builder, fcTool *core.Client) *cobra.C
 					return err
 				}
 			} else {
+				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "\nriff function create: built image %s for service %s.\n", f.Spec.RunLatest.Configuration.RevisionTemplate.Spec.Container.Image, f.Name)
```
Comment (trisberg) [inline pkg/riff/commands/function.go]: should we really hardcode `riff` here?

Comment (fbiville) [reply pkg/riff/commands/function.go]: oops, merged too hastily. A fix is here: https://github.com/projectriff/riff/pull/1206.

Comment (ericbottard) [reply pkg/riff/commands/function.go]: Good catch @trisberg !

**Pull request closed without merging**